xml/System.Windows.Media/LineGeometry.xml

```diff
@@ -24,7 +24,7 @@
 ## Remarks  
  To create multiple connected lines, use a <xref:System.Windows.Media.LineSegment> or <xref:System.Windows.Shapes.Polyline> segment with the <xref:System.Windows.Media.PathFigure> and <xref:System.Windows.Media.PathGeometry> classes.  
   
- **Freezable Features:** Because it inherits from the <xref:System.Windows.Freezable> class, the <xref:System.Windows.Media.LineGeometry> class provides several special features: <xref:System.Windows.Media.LineGeometry> objects can be declared as , shared among multiple objects, made read-only to improve performance, cloned, and made thread-safe. For more information about the different features provided by <xref:System.Windows.Freezable> objects, see [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
+ **Freezable Features:** Because it inherits from the <xref:System.Windows.Freezable> class, the <xref:System.Windows.Media.LineGeometry> class provides several special features: <xref:System.Windows.Media.LineGeometry> objects can be declared as [resources](~/docs/desktop-wpf/fundamentals/xaml-resources-define.md),shared among multiple objects, made read-only to improve performance, cloned, and made thread-safe. For more information about the different features provided by <xref:System.Windows.Freezable> objects, see [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
```

Comment:
```suggestion
 **Freezable Features:** Because it inherits from the <xref:System.Windows.Freezable> class, the <xref:System.Windows.Media.LineGeometry> class provides several special features: <xref:System.Windows.Media.LineGeometry> objects can be declared as [resources](~/docs/desktop-wpf/fundamentals/xaml-resources-define.md), shared among multiple objects, made read-only to improve performance, cloned, and made thread-safe. For more information about the different features provided by <xref:System.Windows.Freezable> objects, see [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
```